_docs/howto/deploy/change-gcp-deployment.md

```diff
@@ -1,28 +1,64 @@
 ---
-title: Changing a Deployment
+title: Updating Forseti
 order: 104
 ---
 #  {{ page.title }}
 
-This page describes how to change a Forseti Security deployment. Most changes
-to a deployment are usually to the deployment properties, such as the
-instance type or `src-path` when you want your deployment to run a certain 
-version of Forseti Security. To update your deployment, follow the process below:
+### Update local installation
+If you want to update your local version of Forseti, pull in the latest changes from
+git:
 
-  1. Edit `deploy-forseti.yaml` and update the values you want to change.
+```bash
+$ cd forseti-security
+$ git pull
+```
+
+And then run the python installation:
+
+```bash
+$ python setup.py install
+```
+
+### Change deployment properties
+If you need to change any of the properties for your Forseti deployment, such as the
+`release-version`, follow the process below:
+
+  1. If you **do not** need to update your Forseti version, continue on to step 3.
+     Sync your local git code (where you ran Deployment Manager) so you get the latest 
+     deployment templates. This is important because a newer version of Forseti might 
+     also require an updated startup script on the GCE instance.
+     
+     To get the latest version of Forseti, sync your master branch:
+     
+     ```bash
+     $ git checkout master
+     $ git pull
+     ```
+     
+  1. Check `deploy-forseti.yaml.sample` to see if there are any new properties 
+     that you need to copy over to your previous `deploy-forseti.yaml`.
+  1. Edit `deploy-forseti.yaml` and update the values you want to change. If you 
+     previously deployed the `master` branch, you don't need to change it.
   1. Run the following update command:
 
       ```bash
       $ gcloud deployment-manager deployments update DEPLOYMENT_NAME \
        --config path/to/deploy-forseti.yaml
       ```
 
-If you change the properties in the `deploy-forseti.yaml` "Compute Engine" 
+If you changed the properties in the `deploy-forseti.yaml` "Compute Engine" 
 section or the startup script in `forseti-instance.py`, you need to reset 
 the instance for changes to take effect:
 
   ```bash
   $ gcloud compute instances reset COMPUTE_ENGINE_INSTANCE_NAME
   ```
 
+The GCE instance will restart and perform a fresh installation of Forseti, so you do 
+not need to ssh to the instance to run all the git clone/python install commands.
+
+There are limitations as to what resources you can update in your deployment. If you 
```

Comment:
Is it likely that I'll need to create a new deployment, or is it a certainty that if I get an error that I can't change a resource, I will have to create a new deployment?

If I have to create a new deployment, do I start from scratch with http://forsetisecurity.org/docs/howto/deploy/gcp-deployment or http://forsetisecurity.org/docs/howto/deploy/local-deployment (could be good to link), or is there a different method?

Reply:
Yes, if you are trying to change an immutable property, deployment update will fail. (see the official docs link below)

Since this is the case of Deployment Manager, the user will either use GCP deployment or the setup wizard.

Reply:
Thanks; given that, I'd want to use more definitive language here. "Some resources can't be updated in a deployment. If you see an error that you can't change a certain resource, you'll need to create a new [GCP deployment]() or [local deployment]()."